src/components/Navigation.vue

```diff
@@ -135,6 +164,12 @@ export default {
     isHomePage() {
       return this.$route.name === 'home';
     },
+    alpes() {
```

Comment:
je pense qu'il y a moyen de faire une grosse factorisation de code ici.
L'idée c'est d'éviter de devoir modifier ce fichier le jour ou on voudra rajouter le jura ou les vosges. L'idéal serait de n'avoir qu'a rajouter le massif dans massifs.json avec un `"mountain_range": "Vosges",` et ce fichier va générer dynamiquement le navigation drawer, sans avoir besoin d'écrire "Vosges" en dur.

Pour ça à priori 2 choses : 
- ici une computed qui retourne un tableau de "mountain_range" (["Alpes, Pyrénées", ...]) à partir de massifs.json: 
```
mountainRanges() {
  return [...new Set(this.massifs.map(massif => massif.location.mountain_range))];
}
```

- et plus haut, dans le template, une loop v-for qui traverse cette computed et génère la <v-list-group> de chaque mountainRange

Reply:
Done ✅